backends/generators/sverilog/sverilog_generator.py

```diff
@@ -0,0 +1,179 @@
+        # For other cases, pad on the right
+        match_str = match_str + "-" * (32 - len(match_str))
+    
+    # Convert to SystemVerilog format (0, 1, or ?)
+    result = []
+    for bit in match_str:
+        if bit == "0":
+            result.append("0")
+        elif bit == "1":
+            result.append("1")
+        else:  # '-' or any other character
+            result.append("?")
+    
+    return "32'b" + "".join(result)
+
+
+def generate_sverilog(instructions, csrs, output_file):
+    """Generate SystemVerilog package file."""
+    with open(output_file, "w") as f:
+        # Write header
+        f.write("\n/* Automatically generated by parse_opcodes */\n")
+        f.write("package riscv_instr;\n")
+        
+        # Find the maximum name length for alignment
+        max_instr_len = max((len(format_instruction_name(name)) for name in instructions.keys()), default=0)
+        max_csr_len = max((len(format_csr_name(csrs[addr])) for addr in csrs.keys()), default=0)
+        max_len = max(max_instr_len, max_csr_len)
+        
+        # Write instruction parameters
+        for name in sorted(instructions.keys()):
+            encoding = instructions[name]
+            sv_name = format_instruction_name(name)
+            # Pad the name for alignment
+            padded_name = sv_name.ljust(max_len)
+            
+            # Get the match pattern
+            if isinstance(encoding, dict) and "match" in encoding:
+                match = encoding["match"]
+            else:
+                # If no match field, use all wildcards
+                match = "-" * 32
+            
+            # Check if this is a compressed instruction
+            is_compressed = name.startswith("c.")
+            sv_bits = match_to_sverilog_bits(match, is_compressed)
+            f.write(f"  localparam [31:0] {padded_name} = {sv_bits};\n")
+        
+        # Write CSR parameters
+        # CSRs are returned as {address: name} by load_csrs
+        for addr in sorted(csrs.keys()):
+            csr_name = csrs[addr]
+            sv_name = format_csr_name(csr_name)
+            # Pad the name for alignment
+            padded_name = sv_name.ljust(max_len)
+            
+            # Format CSR address as 12-bit hex
+            f.write(f"  localparam logic [11:0] {padded_name} = 12'h{addr:03x};\n")
+        
+        # Write footer
+        f.write("\nendpackage\n")
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Generate SystemVerilog package from RISC-V instruction definitions"
+    )
+    parser.add_argument(
+        "--inst-dir",
+        default="../../../gen/resolved_spec/_/inst/",
+        help="Directory containing instruction YAML files",
+    )
+    parser.add_argument(
+        "--csr-dir",
+        default="../../../gen/resolved_spec/_/csr/",
+        help="Directory containing CSR YAML files",
+    )
+    parser.add_argument(
+        "--output", 
+        default="inst.sverilog", 
```

Comment:
What about something like this instead?
```suggestion
        default="riscv_decode_package.svh", 
```